src/rules/at-each-key-value-single-line/index.js

```diff
@@ -102,19 +110,26 @@ function mapName(mapDecl, mapNamespace) {
 // Returns the parameters of a map-get or map.get call
 // Returns [map variable, key_variable]
 function mapGetParameters(mapGetDecl, mapNamespace) {
-  const pattern = getNamespacedPattern("get\\((.*), ?(.*)\\)", mapNamespace);
+  const pattern = getNamespacedPattern(
+    "get\\((.*), ?(.*)\\)",
+    mapNamespace,
+    mapGetDecl
+  );
   const parts = mapGetDecl.match(new RegExp(pattern));
 
   return [parts[1], parts[2]];
 }
 
-function getNamespacedPattern(pattern, mapNamespace) {
-  switch (mapNamespace) {
-    case "*":
-      return pattern;
-    case null:
-      return `map[-.]${pattern}`;
+function getNamespacedPattern(pattern, namespace, decl) {
+  if (namespace === null) {
+    return `map[-.]${pattern}`;
   }
 
-  return `${mapNamespace}[-.]${pattern}`;
+  if (namespace === "*") {
+    return pattern;
+  }
+
+  return new RegExp(`${namespace}.${pattern}`).test(decl)
+    ? `${namespace}.${pattern}`
```

Comment:
this misses escaping the `.`, which would make it match things like `miget` for the `m` namespace